vm-virtio/src/pmem.rs

```diff
@@ -322,13 +322,22 @@ pub struct Pmem {
     interrupt_cb: Option<Arc<dyn VirtioInterrupt>>,
     epoll_threads: Option<Vec<thread::JoinHandle<result::Result<(), DeviceError>>>>,
     paused: Arc<AtomicBool>,
+
+    // Hold ownership of the memory that is allocated for the device
+    // which will be automatically dropped when the device is dropped
+    _region: MmapRegion,
 }
 
 impl Pmem {
-    pub fn new(disk: File, addr: GuestAddress, size: GuestUsize, iommu: bool) -> io::Result<Pmem> {
+    pub fn new(
+        disk: File,
+        addr: GuestAddress,
+        _region: MmapRegion,
```

Comment:
Nit: This does not need the `_` since it's used by the function. Only the structure field requires the `_` because it is used as a placeholder.

Reply:
I left it like that so that I can use the lazy initialisation of the struct :-)